spec/features/conditions/user_creates_condition_spec.rb

```diff
@@ -0,0 +1,48 @@
+require './spec/spec_helper'
+
+describe "user visits /conditions/new" do
+
+  it "user sees create condition message" do
+    visit('/conditions/new')
+
+    expect(page).to have_content("Create a New Condition")
+  end
+
+  it "user sees the Conditions form" do
+    visit('/conditions/new')
+
+    expect(page).to have_content("Date:")
+    expect(page).to have_content("Max Temperature:")
+    expect(page).to have_content("Mean Temperature:")
+    expect(page).to have_content("Minimum Temperature:")
+  end
+
+  it "user can enter a new condition" do
```

Comment:
i think this test verifies the two tests defined above. we don't need to test these all individually